test/unit/alignment/multiple/align_multiple_test.cpp

```diff
@@ -84,3 +111,146 @@ TEST(align_multiple_test, the_first_aminoacid_test)
+    std::vector<seqan3::dna5_vector> input{"NNTGTNN"_dna5, "GGTNTNNGT"_dna5, "NGTNTGGG"_dna5};
+    seqan3::gap g{};
+    std::vector<std::vector<seqan3::gapped<seqan3::dna5>>> output
+    {
+        {'N'_dna5, 'N'_dna5, 'T'_dna5, 'G'_dna5, 'T'_dna5, 'N'_dna5,        g, 'N'_dna5,        g,        g,        g,        g,        g},
+        {'G'_dna5,        g,        g, 'G'_dna5, 'T'_dna5, 'N'_dna5, 'T'_dna5, 'N'_dna5, 'N'_dna5, 'G'_dna5, 'T'_dna5,        g,        g},
+        {       g, 'N'_dna5,        g, 'G'_dna5, 'T'_dna5, 'N'_dna5, 'T'_dna5,        g,        g, 'G'_dna5,        g, 'G'_dna5, 'G'_dna5}
+    };
+
+    constexpr seqan3::configuration cfg = seqan3::align_cfg::gap{seqan3::gap_scheme{seqan3::gap_score{-2},
+                                                                                    seqan3::gap_open_score{0}}};
+
+    auto result = seqan3::align_multiple(input, cfg);
+
+    EXPECT_RANGE_EQ(output, result);
+}
+
+TEST(align_multiple_test, the_third_gap_score_test)
+{
+    //alignment generated with seqan_tcoffee app: ./seqan_tcoffee -s data/dna3.fa -a dna -g -16 -e -4 -o data/out_g3.fa
+    std::vector<seqan3::dna15_vector> input{"GGGTGGYTG"_dna15, "KTGTGGYTYTG"_dna15, "KTGTYYYTG"_dna15};
+    seqan3::gap g{};
+    std::vector<std::vector<seqan3::gapped<seqan3::dna15>>> output
+    {
+        {'G'_dna15, 'G'_dna15, 'G'_dna15, 'T'_dna15, 'G'_dna15, 'G'_dna15,         g,         g, 'Y'_dna15, 'T'_dna15, 'G'_dna15},
+        {'K'_dna15, 'T'_dna15, 'G'_dna15, 'T'_dna15, 'G'_dna15, 'G'_dna15, 'Y'_dna15, 'T'_dna15, 'Y'_dna15, 'T'_dna15, 'G'_dna15},
+        {'K'_dna15, 'T'_dna15, 'G'_dna15, 'T'_dna15, 'Y'_dna15, 'Y'_dna15,         g,         g, 'Y'_dna15, 'T'_dna15, 'G'_dna15}
+    };
+
+    constexpr seqan3::configuration cfg = seqan3::align_cfg::gap{seqan3::gap_scheme{seqan3::gap_score{-4},
+                                                                                    seqan3::gap_open_score{-14}}};
+
+    auto result = seqan3::align_multiple(input, cfg);
+
+    EXPECT_RANGE_EQ(output, result);
+}
+
+TEST(align_multiple_test, gap_score_conversion_test)
+{
+    //alignment with seqan3::align_multiple interface ------------------------------------------------------------------
+    constexpr seqan3::configuration cfg = seqan3::align_cfg::gap{seqan3::gap_scheme{seqan3::gap_score{-1},
+                                                                                    seqan3::gap_open_score{-1}}};
+    std::vector<seqan3::rna4_vector> input{"UUUACCCGAG"_rna4, "UUCCCGCG"_rna4, "UUUACGU"_rna4};
+    seqan3::gap g{};
+    //with go = -1, g = -1
+    auto seqan3_interface_result = seqan3::align_multiple(input, cfg);
+
+    //alignment with seqan2 --------------------------------------------------------------------------------------------
+    using alphabet_type = seqan::Rna;
+
+    using seqan3_alphabet_type = seqan3::rna4;
```

Comment:
This is seqan2 section, I would move this line somewhere else.

Reply:
this line is unnecessary after applying the changes below ( https://github.com/smehringer/seqan3/pull/6#discussion_r466977105) 
seqan3_alphabet_type is not needed anymore.